kore/test/Test/Kore/Step/Simplification/Or.hs

```diff
@@ -268,14 +274,14 @@ stateIntention actualAndSoOn =
     Unparser.renderDefault $ Pretty.vsep ("expected: " : actualAndSoOn)
 
 orChild
-    :: (TestTerm, TestPredicate, TestSubstitution)
-    -> TestPattern
+    :: (TermLike RewritingVariableName, TestPredicate, TestSubstitution)
+    -> Pattern RewritingVariableName
 orChild (term, predicate, substitution) =
     Conditional { term, predicate, substitution }
 
 -- Note: we intentionally take care *not* to simplify out tops or bottoms
 -- during conversion of a Conditional into an OrPattern
 wrapInOrPattern
-    :: (TestTerm, TestPredicate, TestSubstitution)
-    -> OrTestPattern
+    :: (TermLike RewritingVariableName, Predicate RewritingVariableName, Substitution RewritingVariableName)
+    -> OrPattern.OrPattern RewritingVariableName
```

Comment:
This line is really long.

I just noticed: do we use `TestTerm`, `TestPredicate` and `TestSubstitution` anywhere anymore? Would it make sense to make them aliases for `TermLike RewritingVariableName` etc.? I assume you did that for `TestPredicate` and `TestSubstitution` already (I'm looking at `orChild` and I don't think it would compile otherwise), right?

Or maybe we don't want to use the aliases anymore, I don't know what would be best. Maybe in situations like the one here it would make sense to use them for brevity, and in other places we could just use whatever we like more.

Reply:
`type TestPredicate = Predicate VariableName` and `type TestSubstitution = Substitution VariableName` are still used in `Test.Kore.Internal.Substitution`.
`type TestTerm = TermLike VariableName` is still used in `Test.Kore.Internal.TermLike`
`TestPredicate` and `TestSubstitution` were redefined in this module so changing them to use `RewritingVariableName` didn't interfere with other modules.

Now I think it can be confusing, so I'll remove these two aliases from this module.